src/components/ChooseDialect.js

```diff
@@ -137,6 +165,25 @@ const ChooseDialect = ({ navigation, dialects, completedGrammars }) => {
         <TextWrapper>
           <MessageText>More dialects coming soon!</MessageText>
         </TextWrapper>
+        <TextWrapper>
+          <Button
+            mode="contained"
+            onPress={() => signout()}
+            color={"#E9A9BA"}
+            labelStyle={{
+              color: "#fff",
+              fontSize: 18,
+            }}
+            style={{
+              width: 280,
+              height: 45,
+              margin: 10,
+              justifyContent: "center",
+            }}
+          >
+            LOGOUT
```

Comment:
I think we will probably want this logout button somewhere else but it is fine here for now.